app/jobs/recalculate_handover_date_job.rb

```diff
@@ -5,7 +5,7 @@ class RecalculateHandoverDateJob < ApplicationJob
 
   def perform(nomis_offender_id)
     offender = OffenderService.get_offender(nomis_offender_id)
-    if offender&.inside_omic_policy?
+    if offender&.inside_omic_policy? && offender&.has_case_information?
```

Comment:
This should become redundant after MO-628 is merged - prison.offenders changes to only return people 'actively' in the service, so this job can't be executed with an invalid offender

Reply:
@starswan Does something in MO-628 also check for `has_case_information?` – or is that still needed here?

Reply:
This whole ticket becomes redundant. Prison#offenders now returns just the offenders who are 'officially' in the service